docs/contributing/working.md

```diff
@@ -50,17 +48,38 @@ chore: add unit-tests for all components
 fix: update MSFT design system to use 3px border radius
 ```
 
-### Formatting messages
+### Pull reqest template
+
+In addition to providing a properly formatted title for your pull request, we offer a template that will help guide you toward appropriate best practices and processes as well as provide additional information which will give those reviewing your request the context they need.
```

Comment:
Instead of pasting the pull request template files contents below, which we may update and then this page will fall out of date, can we link to the file instead?

Reply:
Fixed.